secretsauce/apps/portal/serializers.py

```diff
@@ -13,7 +13,7 @@ class ProjectSerializer(serializers.ModelSerializer):
 
```

Comment:
Just noticed this. What is line 12 for? Not relevant right?

Reply:
Yup, not relevant, removed.